pex/bin/pex.py

```diff
@@ -808,11 +825,15 @@ def build_pex(
     pex_info.strip_pex_env = options.strip_pex_env
     pex_info.interpreter_constraints = interpreter_constraints
 
-    for requirements_pex in options.requirements_pexes:
-        pex_builder.add_from_requirements_pex(requirements_pex)
```

Comment:
I'm guessing you're aware that this removed call seems to be the only caller of this `add_from_requirements_pex` method?

Reply:
I had missed that in the shuffles. Thanks - killed.